RxCocoa/RxCocoa/iOS/UITextView+Rx.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UITextView+Rx.swift
+//  RxCocoa
+//
+//  Created by Yuta ToKoRo on 7/18/15.
+//  Copyright (c) 2015 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import UIKit
+
+extension UITextView {
```

Comment:
You can also add 
```
    public var rx_delegate: DelegateProxy {
        return proxyForObject(self) as RxTextViewDelegateProxy
    }
```

... like other classes that have delegate have. But I can also add this myself if you don't want to investigate how this works right now.